templates/enabledLDAP.sql

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Change enabledLDAP.sql to another name. This file should be separated into several files (1 per plugin) that run depending on active plugins or else a single template which can be overwritten by the user.

-----------------------------------------------------------------------------------------------------------

Cambiar enabledLDAP.sql por otro nombre. Este archivo debería separarse en varios archivos (1 por plugin) que se ejecutan dependiendo de los plugins activos o más bien un único template que pueda ser sobreescrito desde fuera del rol.

Reply:
Change the SQL statements to the var of the role. I add the peertube_sql_extras variable where these files are defined